charger/ocpp.go

```diff
@@ -397,21 +413,18 @@ func (c *OCPP) currents() (float64, float64, float64, error) {
 	return c.cp.Currents()
 }
 
-// // TODO: Phases1p3p implements the api.PhaseSwitcher interface
-// func (c *OCPP) Phases1p3p(phases int) error {
-// 	if !c.phaseSwitchingSupported {
-// 		return fmt.Errorf("phase switching is not supported by the charger")
-// 	}
+// Phases1p3p implements the api.PhaseSwitcher interface
+func (c *OCPP) phases1p3p(phases int) error {
+	c.phases = phases
 
-// 	err := c.setPeriod(c.current, phases)
-// 	if err == nil {
-// 		c.phases = phases
-// 	}
-
-// 	return err
-// }
+	// NOTE: this will currently _never_ do anything since
+	// loadpoint disabled the charger before switching so
+	// updatePeriod will short-circuit
```

Comment:
Shouldn't be like that?
```suggestion
	// updatePeriod will not short-circuit
```

Reply:
Why? When `phases1p3p` is called, the charger has been disabled before. Hence- not enabled- `updatePeriod` will just return.

Reply:
Another misunderstanding, I thought you meant electrical short-circuit.